docs/trace/advanced/stratified-sampling-example/README.md

```diff
@@ -0,0 +1,98 @@
+# Stratified Sampling: An Example
+
+This is a proof of concept for how we can achieve stratified sampling in
```

Comment:
This is a doc showing how to achieve...

Reply:
Updated with a small tweak - please let me know if the new text is better.